ui/src/router/components/configuration/components/router_config_section/RoutesTableExpandedRow.js

```diff
@@ -25,40 +28,90 @@ const OrDivider = () => (
   </EuiFlexGroup>
 );
 
-export const RoutesTableExpandedRow = ({ item }) => {
+export const RoutesTableExpandedRow = ({ item, inDefaultTrafficRule }) => {
   return (
     <EuiFlexGroup direction="column" gutterSize="s">
       <EuiFlexItem>
-        <EuiCallOut
+        {item.rules && <EuiCallOut
```

Comment:
`item.rules &&` -> `!!item.rules &&` (here and elsewhere in this file) may be better if `rules` is not already boolean.